roles/foreman_provisioning_infrastructure/tasks/main.yml

```diff
@@ -0,0 +1,120 @@
+---
+- name: 'restart foreman-proxy'
+  service: name=foreman-proxy state=restarted
+
+- name: 'restart foreman'
```

Comment:
If the installer option is used this can be removed, otherwise it can be moved to where the package is installed.